flight/PiOS/posix/pios_serial.c

```diff
@@ -45,11 +43,18 @@
 #include <fcntl.h>
 #include <termios.h>
 #include <unistd.h>
+#ifdef PIOS_INCLUDE_SERIAL
```

Comment:
THe meaning of PIOS_INCLUDE_SERIAL is changed so that we can get most of this code (which also handles stdio telemetry etc) on MacOS, but not the Linux specific parts.

Reply:
And windows, which makes the build fail since they don't have the good APIs 😛.